client/package.json

```diff
@@ -832,6 +823,7 @@
   },
   "dependencies": {
     "find": "^0.2.7",
+    "language-server-bitbake": "^2.2.0",
```

Comment:
For packaging changes you must make sure the following work as they used to:
 - debug launch (client/server)
 - .vsix package
 - integration tests
 - integration tests debug launch

Reply:
These are still broken. That's where the real complexity of changing the packaging between client and server lie.